.cspell.json

```diff
@@ -164,7 +164,9 @@
     "WBITS",
     "laddr",
     "nrows",
-    "Dumpable"
+    "Dumpable",
```

Comment:
def start_trace():

   global tracer
   # detect PFS 
   
   session_id
   tracer = (endpoint, context = {experiment, run,sessio}
   print("you can visulaize trace in url")
   

def _get_tracer()
   start_trace if no tracer
   return tracer
   

@trace 
	span = xxx
	tracer.trace(span)

Reply:
what should we expect for parameter `span` in `tracer.trace()`? is it a real OTEL span, we just need to export it? or it's just a dictionary that we need to build the span?